cmd/gotagger/main.go

```diff
@@ -61,6 +61,7 @@ type GoTagger struct {
 	tagRelease     bool
 	versionPrefix  string
 	dirtyIncrement string
+	commitMapfile  string
```

Comment:
Let's name this `configFile`. Pretty much guaranteed it'll hold more than just the commit mapping...

Reply:
Done